api/src/data/migrations/019.create-actions.ts

```diff
@@ -0,0 +1,30 @@
+import * as knex from "knex";
+
+export async function up(knex: knex.Knex) {
+    await knex.schema.createTable("actions", function(table) {
+        table.increments("id").primary().notNullable();
+        table.integer("hazard_id").notNullable();
+        table.integer("creator_employee_id").nullable();
```

Comment:
@burkkyy Anything that ends in `_id` should have a foreign key reference.

Reply:
agreed